apps/aecore/lib/aecore/keys/peer.ex

```diff
@@ -0,0 +1,117 @@
+defmodule Aecore.Keys.Peer do
```

Comment:
can't we use a lot of functionality from the wallet here?

Reply:
Yes, we are going to, but I'll add it in the Sync PR #414

Reply:
yes okay